autotest/gdrivers/kea.py

```diff
@@ -607,4 +607,14 @@ def test_kea_14():
     gdaltest.kea_driver.Delete('tmp/out2.kea')
 
 
+###############################################################################
+# Test /vsi functionality
+
+
+def test_kea_15():
+    if gdaltest.kea_driver is None:
+        pytest.skip()
 
+    ds = gdal.Open('/vsicurl/https://bitbucket.org/chchrsc/kealib/downloads/utm.kea')
```

Comment:
I'd prefer if we avoid network access as much as possible to avoid failures in case of network issues
You could ingest in vsimem a file with something like
```gdal.FileFromMemBuffer('/vsimem/foo.kea', open(some_kea_file, 'rb').read())```
and then gdal.Open() it